### ISSUE TITLE
Segmentation: merge multiple PolygonLabels to one annotation

### ISSUE THREAD

**Issue by JSchuerz** [heartexlabs/label-studio#2766]
It would be nice to be able to 'merge' multiple PolygonLabels to one annotation, e.g. in Coco formate:
segmentation: [[0, 0, 1, 1, 1, 0], [2, 2, 2, 3, 3, 3]] i.e. two unconnected triangles belong to one annotation.

This is especially convenient for Instance Segmentation, where one annotation describes the mask of one instance (which does not need to be connected...)

If this is already implemented, please hint me towards some documentation - but I haven't found any on the website.

**Comment by makseq** [comment]
@JSchuerz no, it's not implemented yet and contributions are highly appreciated! You can do it in the converter module. e.g. in this place: https://github.com/heartexlabs/label-studio-converter/blob/master/label_studio_converter/converter.py#L436
